src/pages/Auth/Signup.jsx

```diff
@@ -0,0 +1,371 @@
+    }
+    if (
+      userInfo.lastName === "" ||
+      !/^[a-zA-Z]+(\s*\w*)*$/.test(userInfo.lastName)
+    ) {
+      errorsDispatch({
+        type: "SET_LASTNAME_ERROR",
+        payload: "Please enter valid surname",
+      });
+      error = true;
+    }
+
+    if (userInfo.email === "" || !/^.+@.+\.com$/.test(userInfo.email)) {
+      errorsDispatch({
+        type: "SET_EMAIL_ERROR",
+        payload: "Please enter valid email id",
+      });
+      error = true;
+    }
+
+    if (
+      userInfo.password === "" ||
+      !/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/g.test(
+        userInfo.password
+      )
+    ) {
+      errorsDispatch({
+        type: "SET_PASSWORD_ERROR",
+        payload:
+          "Password length should contain minimum 8 characters (at least one capital, small letter and number)",
+      });
+      error = true;
+    }
+    if (
+      userInfo.confirmPassword === "" ||
+      userInfo.password !== userInfo.confirmPassword
+    ) {
+      errorsDispatch({
+        type: "SET_RE_PASSWORD_ERROR",
+        payload: "Password does not match",
+      });
+      error = true;
+    }
+    return error;
+  };
+
+  const submitSignupForm = (e) => {
+    e.preventDefault();
+    if (!checkFormValidity()) {
+      postSignupData();
+    }
+  };
+
+  return (
+    <main class="main-container">
+      <div class="form-container">
+        <h4 class="secondary-text-color  text-center">SIGN UP</h4>
+        <div class="title-underline"></div>
+
+        <form class="form-col">
+          <div class="row">
+            <div class="column-30-pc">
+              <label
+                for="first-name"
+                class="form-label text-bold-weight form-label-required-field"
+              >
+                First Name
+              </label>
+            </div>
+            <div class="column-70-pc">
+              <input
+                class="form-field"
+                type="text"
+                placeholder="Enter your first name"
+                name="firstName"
+                onChange={onChangeHandler}
+                onFocus={() => {
+                  onFocusClearError("SET_FIRSTNAME_ERROR");
+                }}
+                required
+              />
+              <div
+                class="form-validation-msg"
+                style={{
+                  display: fieldErrors.firstnameError ? "block" : "none",
+                }}
+              >
+                <span>
+                  <i class="fas fa-exclamation-circle"></i>
+                </span>
+                {fieldErrors.firstnameError}
+              </div>
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="column-30-pc">
+              <label
+                for="first-name"
+                class="form-label text-bold-weight form-label-required-field"
+              >
+                Last Name
+              </label>
+            </div>
+            <div class="column-70-pc">
```

Comment:
These class should be changed with className

Reply:
Done